### PR TITLE
[FIX] website_blog: change blog cover title to h1

### PR DESCRIPTION
**Pull request by xO-Tx**
SEO issue: blog post doesn't have `<h1>` for post title.
The goal of this PR is to change "first cover" title
to `<h1>`.
"Next blog" cover title will have a h1 style too.

task-2454130

### CODE REVIEW

**Review by qsm-odoo — COMMENTED** [review]
My questions are rather for you @JKE-be here :)

File: addons/website_blog/views/website_blog_templates.xml [hunk 1]
```diff
@@ -443,7 +443,7 @@ list of filtered posts (by date or tag).
 
                         <div class="container text-center position-relative h-100 d-flex flex-column flex-grow-1 justify-content-around">
                             <div t-attf-class="o_wblog_post_title">
-                                <div t-field="next_post.name" id="o_wblog_post_name" t-att-data-blog-id="next_post.id" placeholder="Blog Post Title"/>
+                                <div t-field="next_post.name" id="o_wblog_post_name" t-att-data-blog-id="next_post.id" placeholder="Blog Post Title" class="h1"/>
```
[the remark this answers is not in the document]
Comment (JKE-be) [reply addons/website_blog/views/website_blog_templates.xml:446]: Yes I confirm, my point was to don't add tag h1 in the 'next article' like previously (in v8)
Not especially to add class='h1' in the 'next article' ;) So just need to check class vs font family as explain in edit message of qsm

File: addons/website_blog/views/website_blog_templates.xml [hunk 2]
```diff
@@ -240,7 +240,7 @@ list of filtered posts (by date or tag).
 
                     <div class="container text-center position-relative h-100 d-flex flex-column flex-grow-1 justify-content-around">
                         <div t-attf-class="o_wblog_post_title #{opt_blog_post_select_to_tweet and 'js_tweet'} #{opt_blog_post_select_to_comment and 'js_comment'}">
-                            <div t-field="blog_post.name" id="o_wblog_post_name" data-oe-expression="blog_post.name" t-att-data-blog-id="blog_post.id" placeholder="Blog Post Title"/>
+                            <h1 t-field="blog_post.name" id="o_wblog_post_name" data-oe-expression="blog_post.name" t-att-data-blog-id="blog_post.id" placeholder="Blog Post Title"/>
```
[the remark this answers is not in the document]
Comment (JKE-be) [reply addons/website_blog/views/website_blog_templates.xml:243]: not in stable...
I have myself a module that replace this div to an h1 ;)
So after the next update my website will be crashed

**Comment by JKE-be** [comment]
So go for master imo

**Comment by qsm-odoo** [comment]
Ok thanks @JKE-be 

@xO-Tx 

So:
- master
- remove the font-family css rule you added
- keep the h1 class you added (to get the correct font family)

**Review by qsm-odoo — APPROVED** [review]
@robodoo r+

**Review by qsm-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**